src/components/structures/MatrixChat.js

```diff
@@ -655,7 +667,7 @@ module.exports = React.createClass({
             }
         }
         else {
-            if (screen) console.error("Unknown screen : %s", screen);
+            console.error("Unknown screen : %s", screen);
```

Comment:
Yes. I don't feel that we should be suppressing this message if `screen` is `undefined`. We should *never* be calling it with `undefined`.